doc/extending.rst

```diff
@@ -128,13 +128,22 @@ config plugin
 
 To enable a plugin create a section named after the plugin category.
 The value is an entry point to the python class/function/object
-that implements the plugin. The format is <module-name>:<attribute-name>.
+that implements the plugin. The format is `<module-name>:<attribute-name>`.
 
-Example of command plugin implemented in the *class* `FooCmd`,
-located at the module `my_plugins.py`::
+Example of command plugin configured in `pyproject.toml`, implemented in
+the *class* `FooCmd`, located at the module `my_plugins.py`:
 
- [COMMAND]
- foo = my_plugins:FooCmd
+.. code-block:: toml
+
+   [tools.doit.plugins]
+   COMMAND = { foo = "my_plugins:FooCmd" }
```

Comment:
what about? 

```
[doit.plugins.foo]
kind = "command"
obj = "my_plugins:FooCmd"
```

Better name for "obj" ?

Reply:
yeah, naming things is hard. If it was purely data, the more verbose form would feel good, but i think in this case spareness wins.

I believe this could also be written

```toml
[doit.plugins.COMMAND]
foo = "my_plugins:FooCmd"
```

(have updated docs to reflect both)